### DISCUSSION TITLE
Auto add new issues to a project

### DISCUSSION THREAD

**Discussion by geekpulp** [community/community#5713] [Projects and Issues] [answered]
It would be good if new issues on a repo were added to the project automatically. It would make it easier to do triage of new issues and onboard work into a project.

**Comment by dariota** [comment]
I would add to this the ability to limit the automatic adding only to issue created (or later labels) with specific labels - from a glance at the beta, this looks like the main thing missing to replace the custom automation I have in place on some projects.

**Reply by atorres-eqrx** [reply]
I'm in a similar boat. We have a mono repo and use labels with namespaces e.g. `domain1:infrastructure`, `domain1:database`, `domain2:frontend` to identify the different kinds of tickets within the repo, each serviced by a different team.

Ideally, I would like to have labels matching the pattern of `domain1:*` show up in one project and `domain2:*` in another project. 
And all issues `*` to show up in a comprehensive board.

**Reply by vcaballero-salle** [reply]
I think this can be made with project automations: https://docs.github.com/en/issues/trying-out-the-new-projects-experience/automating-projects.

However,  I think there should be a fixed set of "workflows" that you can set from the project, at least something like automatically add issues and PRs of that repo to this project.

**Comment by alexey-igrychev** [comment]
+PRs

**Comment by alexcnichols** [comment, marked as answer]
👋🏼 Improvements for auto adding issues to projects (beta), and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/286 :octocat: 

Marking this as a duplicate, in favor of this other discussion:

- https://github.com/github/feedback/discussions/5378

**Reply by mesqueeb** [reply]
@alexcnichols when you say "coming soon" how many years are we talking about? It's been more than a year.

**Reply by Mdshoyonmia** [reply]
3143 Al Basatin
6886
Ar Rimal
Riyadh 13438
SA

**Comment by LittleHuman1** [comment]
lexcnichols
on Oct 27
👋🏼 Improvements for auto adding issues to projects (beta), and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. github/roadmap#286 :octocat:

Marking this as a duplicate, in favor of this other discussion:

Automatically add items from repositories (projects beta) #5378

**Comment by FaGru3n** [comment]
Hi together, 👋🏼 

found out that there is a limit for auto-add workflows within an organisation based on the billing account. 
https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically

Do you know a way to delete not needed auto-add workflows within projects? So that we only have the needed auto-add workflows?

Beside appreciation, cool new features within projects! 😉 

Thx for helping us out!
